test/util.ts

```diff
@@ -57,7 +68,7 @@ export function testEachVersion<T extends TestBuilder>(
         const testName = name === undefined ? version : `${name} [${version}]`;
         defineTest(testName, () => {
             const builder = common();
-            builder.setOptions({ luaTarget: version });
+            builder.setOptions({ luaTarget: version, luaLibImport: tstl.LuaLibImportKind.Inline });
```

Comment:
Why, this was require for performance reasons

Reply:
Three tests are failing to import lualib if that option is removed, as I noted earlier. Something to do with the large size of the bundle, as best as I can tell? So switching to inline imports was my solution. Guess I'll have to come up with another answer...

Reply:
These failures are fixed with https://github.com/TypeScriptToLua/TypeScriptToLua/pull/1263/commits/e0d6206ba44de81a8455103999b55236dac30371 and https://github.com/TypeScriptToLua/lua-wasm-bindings/pull/5.

Reply:
Those failures are fixed with https://github.com/TypeScriptToLua/TypeScriptToLua/pull/1263/commits/e0d6206ba44de81a8455103999b55236dac30371 and https://github.com/TypeScriptToLua/lua-wasm-bindings/pull/5. I have reverted this change.